docker-compose.yaml

```diff
@@ -4,7 +4,7 @@ services:
     container_name: idw-keria
     restart: unless-stopped
     build:
-      context: https://github.com/WebOfTrust/keria.git#c9d35f901e293f166d5abbd58cda249248555253
+      context: https://github.com/WebOfTrust/keria.git#736947191069cfc1617a43f63dbc64210c351b5c
```

Comment:
There's 3 places in our repo that has the githash to be updated

(I think we can consolidate the 2 docker compose files eventually, just nice to have the traefik stuff in case I lose our devbox)

Also, `chore` or `build` instead of `fix` ;)